### DISCUSSION TITLE
Can't use as props in Controller

### DISCUSSION THREAD

**Discussion by fany-lin** [react-hook-form/react-hook-form#4871] [Q&A] [answered]
I want to implement the Radio Group with Material UI and react-hook-form. When I use render in the Controller everything works fine, but when I change render to as it's failed.

Here is my code:
https://codesandbox.io/s/priceless-tdd-93oib?file=/src/pages/FirstPage/FirstPage.tsx:3008-3823

**Comment by tt4g** [comment, marked as answer]
There is no longer support `as` prop of `Controller` on V7, use `render` prop instead.
See #3732

**Reply by fany-lin** [reply]
Oh, I get it. Thanks!
